src/dmclock/sim/src/test_dmclock_main.cc

```diff
@@ -232,9 +237,9 @@ int main(int argc, char* argv[]) {
 
 
 void test::client_data(std::ostream& out,
-		 test::MySim* sim,
-		 test::MySim::ClientFilter client_disp_filter,
-		 int head_w, int data_w, int data_prec) {
+		       test::MySim* sim,
```

Comment:
tabs left

Reply:
Is there an issue with tabs? Many (most? all?) ceph source files have headers of:
> // -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
> // vim: ts=8 sw=2 smarttab

which directs text editors such as vim and emacs to use tabs.